src/frontend/js/modules/attachment/list/attachment-list.controller.js

```diff
@@ -1,12 +1,12 @@
-require('./attachment-list-providers.service.js');
+const _ = require('lodash');
```

Comment:
I think now that we're using Webpack, we can utilize tree shaking. The whole lodash library is quite huge. What we really use is just a fraction of it, but we're currently importing the whole library. The same goes for other utility libraries. I know now is not the time to do optimizations, but this is something we should keep in mind until then.

Reply:
Good point Huy. However we have first to update our lodash to a more recent version, because our version does not support per-method inclusion. and in more recent versions they renamed "contains" to "includes". And restangular uses "contains", as well as parts in our own code...

Reply:
Yes, already tried to have newer version, Restangular failed on it.